### DISCUSSION TITLE
Is it possible to put an image inside a node?

### DISCUSSION THREAD

**Discussion by BONG8** [xyflow/xyflow#4443] [Q&A] [answered]
Hello,
I am trying to put an image inside a custom node but it doesn't seem to work, the image doesn't appear in the inspector.
I'm wondering if it's even possible to put an image but I haven't found no response by searching

**Comment by bcakmakoglu** [comment, marked as answer]
Of course that's possible. Nodes are just `<div>` elements after all.

[Here's an example.](https://codesandbox.io/p/sandbox/withered-rain-h78mdh?file=%2Fsrc%2Fnodes%2FPositionLoggerNode.tsx%3A22%2C9)

**Reply by BONG8** [reply]
Thanks, it was just a bug of my code
